src/cljc/athens/common_db.cljc

```diff
@@ -280,12 +279,12 @@
   "Find and replace linked ref with new linked ref, based on title change."
   [linked-refs old-title new-title]
   (map (fn [{:block/keys [uid string] :node/keys [title]}]
-         (let [[string kw] (if title
+         (let [[text kw]   (if title
                              [title :node/title]
                              [string :block/string])
-               new-str (string/replace string
-                                       (patterns/linked old-title)
-                                       (str "$1$3$4" new-title "$2$5"))]
+               old-wrapped (str "[[" old-title "]]")
+               new-wrapped (str "[[" new-title "]]")
+               new-str     (string/replace text old-wrapped new-wrapped)]
```

Comment:
I think you also need to replace the the `#name-here` format here. Our parser recognizes plain `#alex` as well as `#[[alex]]` and renders both as a ref.

Reply:
On the web preview version I made these three bullets:
```
- #alex
- #[[alex]]
- [[alex]]
```

Then renamed the page from `alex` to `alexxx`, and ended up with these:
```
- #alex
- #[[alexxx]]
- [[alexxx]]
```

Reply:
that plus one more case: `alex` to `al ex` is handled
thanks for spotting it

Reply:
@filipesilva last commit fixes it